apps/src/templates/projects/FeaturedProjectsTable.jsx

```diff
@@ -296,7 +297,7 @@ class FeaturedProjectsTable extends React.Component {
         style={tableLayoutStyles.table}
       >
         <Table.Header />
-        <Table.Body rows={sortedRows} rowKey="channel" />
+        <Table.Body rows={sortedRows} rowKey="projectName" />
```

Comment:
Project names often end up being "My project" or "Remixed: My project" so it channel or even an iterator might be a better unique id.

Reply:
+1, you should not use the project name, but could use anything unique and "stable". You will need the encrypted channel id (sometimes just called channel id) here in order to create a link to your project, so that might be a convenient thing to reuse for this purpose.

For background, reactabular doesn't make it super clear that this should be unique, but it does [say](https://www.npmjs.com/package/reactabular-table#tablebody) that it "allows React to render in a more performant way", which suggests it is probably passing it as the react [key](https://reactjs.org/docs/lists-and-keys.html#keys) which must be unique and stable. "stable" means the same id always points to same project, e.g. not just an index into your current list of items.